streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/RackAwareTaskAssignor.java

```diff
@@ -61,19 +59,30 @@ boolean canMove(final ClientState source,
                         final Map<UUID, ClientState> clientStateMap);
     }
 
+    @FunctionalInterface
+    public interface GetCostFunction {
```

Comment:
`GetCostFunction` -> `CostFunction`

(This interface does not _return_ a cost function, but _is_ a cost function, right?)